### PR TITLE
Feat/pm 704 GitHub config

### PR DESCRIPTION
**Pull request by skrimusss**
[PM-704](https://playmakerpro.atlassian.net/browse/PM-704)

### Task Description
Creating a config for the project

### CODE REVIEW

File: client/next.config.js [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+
+	webpack: (config) => {
+		config.module.rules.push({
+			test: /\.svg$/i,
+			issuer: /\.[jt]sx?$/,
+			use: ['@svgr/webpack'],
+		}),
+			(config.experiments = { ...config.experiments, ...{ topLevelAwait: true } })
+		return config
+	},
```
Comment (dejwi) [inline client/next.config.js:29]: why do we need this ?

File: client/tailwind.config.js [hunk 2]
```diff
@@ -11,83 +11,138 @@ module.exports = {
...
+				spacing9: '3rem',
+				spacing10: '4rem',
+				spacing11: '6rem',
+			},
+			padding: {
+				spacing4: '0.75rem',
+				spacing5: '1rem',
+				spacing6: '1.25rem',
+			},
+		},
```
Comment (dejwi) [inline client/tailwind.config.js:145]: everything I've said above and we don't want to split it between margin/padding
```
spacing: {
	1: '0.125rem',
	2: '0.25rem',
	3: '0.5rem',
        ....
        ....

},
```

**Review by dejwi — CHANGES_REQUESTED** [review]
please look again into this default config and analyse how everything (of what we use) should be defined, don't repeat names:) https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L7

image

not to mention I had to comment out whole `boxShadow` section to get intellisense working, of what I've seen tailwind doesn't support double nesting in config like you did in `boxShadow` and `spacing`, look at linked config for reference


and please use pull request labels, and describe what changes you've made in particular pr, `Creating a config for the project` could mean a lot of things not to mention `... GitHub config` title 😅


tough first tickettttt you got this bro🙏

File: client/next.config.js (hunk 1 above)
Comment (skrimusss) [reply client/next.config.js:29]: this allows to create a react component involving a clean svg

File: client/tailwind.config.js (hunk 2 above)
Comment (skrimusss) [reply client/tailwind.config.js:145]: done

File: client/tailwind.config.js [hunk 3]
```diff
@@ -11,83 +11,115 @@ module.exports = {
 			largeDesktop: '1600px',
 		},
 		colors: {
-			transparent: 'transparent',
-			// primary
+			transparent: {
+				transparent: 'transparent',
+			},
```
Comment (dejwi) [inline client/tailwind.config.js:16]: you've missed one:D I think we can just remove it as tailwind by default has added transparent color

**Review by dejwi — CHANGES_REQUESTED** [review]
When you're done making changes please always click this on the right side 
image



and next time as well please use labels (right side), when we'll start working normally the pull requests tab will be really messy
image

**Review by skrimusss — COMMENTED** [review]
fixed a few things after cr

**Pull request closed without merging**